portablemc/cli/lang.py

```diff
@@ -96,6 +98,7 @@ def get(key: str, **kwargs) -> str:
     "args.start.version.comp.babric": "Start Babric mod loader with beta 1.7.3.",
     "args.start.version.comp.forge": "Start Forge mod loader with latest release.",
     "args.start.version.comp.neoforge": "Start NeoForge mod loader with latest release.",
+    "args.start.version.comp.optifine": "Start Optifine mod loader with latest recommended release.",
```

Comment:
To actually use this, yiou should add `"optifine"` to the tuple in `parse.py`: `for loader in ("fabric", "quilt", "legacyfabric", "babric", "forge", "neoforge"):`.